UsbIpServer/ConnectedClient.cs

```diff
@@ -134,12 +134,36 @@ async Task HandleRequestImportAsync(CancellationToken cancellationToken)
                     RegistryUtils.SetDeviceAsAttached(exportedDevice);
                     var iPEndPoint = ClientContext.TcpClient.Client.RemoteEndPoint as IPEndPoint;
                     RegistryUtils.SetDeviceAddress(exportedDevice, iPEndPoint!.Address.ToString());
+
+                    // Poll for the device to be unplugged
```

Comment:
I'm very open to suggestions if there's a better way to do this than polling. The problem I'm trying to solve is that if the device is silent on the bus, the server never notices that it's missing even if it's unplugged, which can cause incorrect results in the `list` command.

Reply:
Changed to use WMI as you suggested.